examples/step-5/step-5.cc

```diff
@@ -113,6 +113,14 @@ double coefficient (const Point<dim> &p)
   else
     return 1;
 }
+// @note It looks like we use a discontinuous coefficient, however,
+// the finite element method using quadrature really sees only a
+// polynomial approximation (defined by a
+// <code>quadrature_formula</code>) of that discontinuous coefficient
+// on every mesh. (Though that polynomial approximation of course
+// tends to a discontinuous function under mesh refinement.) Refer to
+// CellAccessor::set_material_id to set a discontinuous coefficient
```

Comment:
I suppose here on needs to distinguish between a discontinuous function and one that does not have a one-to-one correspondence. This `coefficient` function is definitely 1-1 (and discontinuous at $r^{2} = 0.5^{2}$), and I agree that this coefficient is effictively "smudged" in cells with quadrature points that evaluate to both sides of the discontinuity. But, @kostyfisik, are you specifically trying to make a point of how you would deal with the situation of no 1-1 correspondence? I don't think that comes across from what you've written here.